pom.xml

```diff
@@ -78,8 +78,7 @@
 
         <!-- Require at least Java 17 to compile -->
```

Comment:
Guess this comment ain't true no more

Reply:
It's kind of true. You'd need Java 17 to run any tests as Spring 6 requires Java 17. With that we require Java 17 for the build, but we compile down to Java 11.

Reply:
Ah that makes sense, thanks

Reply:
Yes this comment was kept on purpose: Java 17 is still necessary to build the project.